src/libraries/System.IO.Compression/src/System/IO/Compression/ZipArchiveEntry.Unix.cs

```diff
@@ -7,6 +7,20 @@ public partial class ZipArchiveEntry
     {
         internal const ZipVersionMadeByPlatform CurrentZipPlatform = ZipVersionMadeByPlatform.Unix;
 
+        /// <summary>
+        /// The default external file attributes are used to support zip archives on multiple platforms.
+        /// For Unix the default external attributes are an equivalent of (Convert.ToInt32("644", 8) &lt;&lt; 16)
+        /// where 644 are rw-r--r-- unix file permissions.
+        /// </summary>
+        internal const uint DefaultFileExternalAttributes = 420 << 16;
```

Comment:
@derwasp can you use the `UnixFileMode` that got just merged, and use the same style as:

https://github.com/dotnet/runtime/blob/b869f40132d9f152bdc46b5a760953579843bae9/src/libraries/System.Formats.Tar/src/System/Formats/Tar/TarHelpers.cs#L23-L24

@carlossanlop @eerhardt @danmoseley should we try to put these consts in a file common to the `Tar` and `Zip` implementations to ensure they use the same values?

Reply:
@tmds Yeah, I just saw that your PR went in, awesome timing! Looking at the API right now.